src/browser-polyfill.js

```diff
@@ -491,19 +491,24 @@ if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.
       get: {minArgs: 1, maxArgs: 1},
       set: {minArgs: 1, maxArgs: 1},
     };
-    apiMetadata.privacy = {
-      network: {
-        networkPredictionEnabled: settingMetadata,
-        webRTCIPHandlingPolicy: settingMetadata,
-      },
-      services: {
-        passwordSavingEnabled: settingMetadata,
-      },
-      websites: {
-        hyperlinkAuditingEnabled: settingMetadata,
-        referrersEnabled: settingMetadata,
-      },
-    };
+
+    apiMetadata.privacy = {};
+    for (const type of ["network", "services", "websites"]) {
+      Object.defineProperty(apiMetadata.privacy, type, {
```

Comment:
Add `configurable: true,` to the properties of the parameter to `Object.defineProperty`, to ensure that the property can be replaced later.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty

Reply:
done.